papis/commands/tag.py

```diff
@@ -0,0 +1,193 @@
+  let you pick the document from those matched (just as with Papis' other
+  commands).
+
+  Tags are only added if they do not already exist, which is to say that the
+  same tag cannot exist more than once in a given document.
+
+- To remove a tag use the `--remove` (or `-r`) option:
+
+    .. code:: sh
+
+        papis tag --remove TAG1 QUERY
+
+  This removes TAG1 from the document. If it didn't exist before, nothing
+  happens. You can use `--remove` as many times as you like to remove multiple
+  tags.
+
+- Use `--drop` (or `-d`) to remove all tags:
+
+    .. code:: sh
+
+        papis tag --drop --new TAG1 QUERY
+
+  This removes all tags from the document and then adds TAG1. You can of course
+  also use `--drop` by itself to simply remove all tags without adding new
+  ones.
+
+- Use `--all` (or `-a`) with any of the other options to apply the tagging
+  operation to all matching documents:
+
+    .. code:: sh
+
+        papis tag --all --new TAG1 QUERY
+
+  This adds TAG1 to all documents matching the QUERY rather than opening the
+  picker to let you choose one.
+
+"""
+from typing import List, Optional, Tuple, Any
+
+import click
+
+import papis.cli
+import papis.commands
+import papis.importer
+import papis.commands.update
+import papis.commands.doctor
+import papis.config
+import papis.document
+import papis.strings
+import papis.tui.utils
+
+logger = papis.logging.get_logger(__name__)
+
+
+@click.command("tag")
+@click.help_option("--help", "-h")
+@papis.cli.git_option()
+@papis.cli.bool_flag(
+    "-d",
+    "--drop",
+    help="Drop all tags.",
+    default=False,
+)
+@click.option(
+    "-p",
+    "--add",
+    "to_add",
+    help="Add a tag.",
+    multiple=True,
+    type=str,
+)
+@click.option(
+    "-r",
+    "--remove",
+    "to_remove",
+    help="Remove a tag.",
+    multiple=True,
+    type=str,
+)
+@click.option(
+    "-n",
+    "--rename",
+    "to_rename",
+    help="Rename a tag (<OLD-TAG NEW-TAG>).",
+    multiple=True,
+    type=(str, str),
+)
+@papis.cli.doc_folder_option()
+@papis.cli.all_option()
+@papis.cli.sort_option()
+@papis.cli.query_argument()
+def cli(
+    git: bool,
+    drop: bool,
+    to_add: List[str],
+    to_remove: List[str],
+    to_rename: List[Tuple[str, str]],
+    sort_field: Optional[str],
+    sort_reverse: bool,
+    query: str,
+    doc_folder: Tuple[str, ...],
+    _all: bool,
+) -> None:
+    """
+    Change a document's tags.
+    """
+
+    # if do_tag:
+    documents = papis.cli.handle_doc_folder_query_all_sort(
+        query, doc_folder, sort_field, sort_reverse, _all
+    )
+
+    if not documents:
+        logger.warning(papis.strings.no_documents_retrieved_message)
+        return
+
+    run_append = papis.commands.update.run_append
+    run_remove = papis.commands.update.run_remove
+    run_drop = papis.commands.update.run_drop
+    run_rename = papis.commands.update.run_rename
+    run_update = papis.commands.update.run
```

Comment:
```suggestion
	from papis.commands.update import (
		run_append, run_remove, run_drop, run_rename, run)
```
? Looks nicer and more python-y to me.